Source/Core/Duality/Utility/Coroutines/CoroutineManager.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Duality.Utility.Pooling;
+
+namespace Duality
+{
+	public class CoroutineManager
+	{
+		// shared pool between all CoroutineManagers (one per scene)
+		private static QueuedPool<Coroutine> pool = new QueuedPool<Coroutine>();
+
+		private readonly Queue<Coroutine> coroutines = new Queue<Coroutine>(256);
+		private readonly Queue<Coroutine> nextCycle = new Queue<Coroutine>(256);
+		private readonly Queue<Coroutine> scheduled = new Queue<Coroutine>();
+		private readonly Queue<Coroutine> trashcan = new Queue<Coroutine>();
+
+		private readonly Dictionary<Coroutine, Exception> lastFrameErrors = new Dictionary<Coroutine, Exception>();
+
+        private int count;
```

Comment:
tabs and spaces are being mixed here

Reply:
Checked all files for mixed whitespace again